libr/bin/dwarf.c

```diff
@@ -194,14 +308,70 @@ static const char *dwarf_langs[] = {
 	[DW_LANG_Fortran08] = "Fortran08"
 };
 
+static const char *dwarf_unit_types[] = {
```

Comment:
```
../libr/bin/dwarf.c:311:20: warning: ‘dwarf_unit_types’ defined but not used [-Wunused-variable]
  311 | static const char *dwarf_unit_types[] = {
      |                    ^~~~~~~~~~~~~~~~
```

Reply:
Yeah, DWARF 5 thing, I am going to add it to the printouts so it would stop complaining for not using it